spec/fixtures/path1

```diff
@@ -0,0 +1 @@
+njsafnjsakasnfkjasnkjnjkfna

```

Comment:
It will be nice to have a better test case. Some text with several paragraphs will be better, as it will test a more sophisticated input data scenario.